apis/core/lib/domain/table/Table.ts

```diff
@@ -55,8 +66,26 @@ export default function mixin<Base extends Constructor<Table>>(Resource: Base):
       return columnMapping
     }
 
-    addColumnMappingFromColumn({ store, column }: CreateColumnMappingFromColumn): ColumnMapping.ColumnMapping {
-      return this.addColumnMapping({
+    addReferenceColumnMapping({ store, targetProperty, referencedTable, identifierMappings }: CreateReferenceColumnMapping): ColumnMapping.ReferenceColumnMapping {
+      const columnMapping = ColumnMapping.createReference(store.create(id.columnMapping(this, targetProperty.value)), {
```

Comment:
I'm not sure about using the target property to build the identifier.

Reply:
You expect it would cause problems?

Reply:
I don't. But then we may as well use `targetProperty` for all column mappings?